development/language/validation.rst

```diff
@@ -271,4 +271,4 @@ License
 * All translations must be released under
   `GNU General Public License 2.0 <http://www.opensource.org/licenses/gpl-2.0.php>`_
 
-.. _Customisation Database: https://www.phpbb.com/go/customise/language-packs
+.. _Customisations Database: https://www.phpbb.com/customise/db/language_packs-25
```

Comment:
At least for 3.1 we have go-links, which are working fine. 

So please use https://www.phpbb.com/go/customise/language-packs/3.1 

@michaelcullum Can we add a go/-link for 3.2 as well?